mycroft/skills/skill_manager.py

```diff
@@ -30,6 +31,51 @@
 SKILL_MAIN_MODULE = '__init__.py'
 
 
+class SettingsMetaQueue:
```

Comment:
I think it would make more sense to put this in settings.py.

Reply:
That's where it's originally was located but I feel like since it's nothing that needs to be shared and only is a helper class for the skill_manager it's better to keep it close to where it's actually used.

Reply:
I think it is more important to group code by function than by where it is used.  Helps discoverability, IMO

Reply:
To me it doesn't make sense, it's more related to the skill manager than the settings system. In the settings.py it would just add confusion.

Reply:
Maybe it should be renamed...

Reply:
Renamed it UploadQueue